packages/preview-button/admin/src/components/PreviewLink/index.css

```diff
@@ -0,0 +1,18 @@
+.utrecht-preview-dialog {
+  max-block-size: 70vh;
+  overflow-y: auto;
+  & .utrecht-preview-dialog__body {
+    --utrecht-space-around: 1;
+  }
+}
+
+.utrecht-preview-link {
+  inline-size: 100%;
+  justify-content: center;
+}
+
+body * div[role="dialog"][aria-labelledby="preview-dialog-label"] {
+  margin-block-start: 32px !important;
+  margin-inline: auto !important;
+  max-width: 100% !important;
+}
```

Comment:
1. Kunnen we niet consistent CSS logical properties gebruiken, dus `max-inline-size` ipv `max-width`?
2. Is `!important` niet al voldoende? Moet de CSS selector echt `body *` bevatten?